### ISSUE TITLE
Provide an option to generate a 'full' rule sheet

### ISSUE THREAD

**Issue by bithalver** [504rules/504rules.github.io#18]
Not a bug, but an enhacement idea: for beginners, it could be a lot of help to list all needed components not just Special game parts.

Example: world 513, you play with residents, so you need some player-number-dependent residents (2p: 30, 3p: 25, 4p: 20, 5stock: 15).

Also, AFAIK all game with residents needs all settlements (game 513 surely needs them and they are not listed).

**Comment by Arachnid** [comment]
I think this is best done as part of any effort to have a 'full' rule sheet mode. I agree it could be useful.

**Comment by bithalver** [comment]
Yep, agree !

But then, we can add a lot more things:
- A sign somewhere on the top if we are playing **with** or **without** residents (you know, the introduction game 123 is **without** residents, and when you first move to a game **with** residents it is a big change).
- Buying new residents at the end of Phase 3C (in general: include all the mandatory phases and their steps).

**Comment by Arachnid** [comment]
The first one is already open as #11, and the second one is already open as #9, although I think that one should be part of this issue and not done on its own.

**Comment by bithalver** [comment]
Another enhancement idea, which could possibly belong here: provide the graphic for the used map. I know, after a lot of plays you can remember the shape of the map by its number, but its a big help not to search for it in the rulebook. Sure it is enough to have a small graphics, for example between Game preparation and the list of Phases (usually we have some graphics there, don't we ? ;) ).

My global idea is to have that kind of two pages where you do not need to read the rulebook for additional info (at least for setup, phases and vvictory points, the later looks very OK for me).

**Comment by Arachnid** [comment]
I agree that'd be useful in an expanded ruleset. There's no way it'll fit in two pages, though - some of the current 'compact' rulesets don't!

**Comment by Engelberg** [comment]
Also, as part of the "full ruleset", it would be a good idea to incorporate the rules which are common to all games, which as far as I can tell, are currently omitted.

A good way to measure the quality of the full ruleset mode would be to compare the quality of the output of module 123 to the in-depth ruleset provided for that combination in the manual.  if all the generated rulesets were of that quality, the app would be dramatically more useful (although it is already a good resource).

**Comment by someknave** [comment]
I will happily add the content, I wouldn't want to mess with the structure that you've laid out. If you let me know how you want additional rules to be tagged I can start getting to work on this.
